src/jni/class_loader.cpp

```diff
@@ -0,0 +1,81 @@
+#include "class_loader.h"
+#include <cassert>
+
+ClassLoader::ClassLoader(): class_loader{jni::JObject()} {
+
+}
+
+jni::JObject to_java_url(jni::Env& env, const String& bootstrapJar) {
+    jni::JClass cls = env.find_class("java/io/File");
```

Comment:
I know this was this way before in GDKotlin, but maybe that's the good time to make use of uniform init.

Reply:
even uglier